ports/libpng/portfile.cmake

```
@@ -77,18 +85,22 @@ vcpkg_cmake_configure(
         -DPNG_SHARED=${PNG_SHARED}
         -DPNG_FRAMEWORK=OFF
         -DPNG_TESTS=OFF
-        -DSKIP_INSTALL_PROGRAMS=ON
         -DSKIP_INSTALL_EXECUTABLES=ON
         -DSKIP_INSTALL_FILES=OFF
+        ${FEATURE_OPTIONS}
     OPTIONS_DEBUG
         -DSKIP_INSTALL_HEADERS=ON
     MAYBE_UNUSED_VARIABLES
         PNG_ARM_NEON
 )
 vcpkg_cmake_install()
+vcpkg_cmake_config_fixup(PACKAGE_NAME PNG CONFIG_PATH lib/cmake/PNG)
```

```suggestion
vcpkg_cmake_config_fixup(PACKAGE_NAME png CONFIG_PATH lib/cmake/PNG)
```
It is just used for the target dir name, lower case is preferred for share in vcpkg, and cmake accepts lower case.